comp/metadata/packagesigning/README.md

```diff
@@ -0,0 +1,58 @@
+# Package Signing Payload
+
+This component populates the Linux Packages signing information in the `inventories` product in DataDog. They fill the `host_signing` table.
+
+This is enabled by default but can be turned off using `inventories_enabled` config.
```

Comment:
I believe it can also be turned off with the new specific config option too if I'm reading it right (`enable_signing_data`)